src/browser/disabled_frame_group_tracker.ts

```diff
@@ -152,45 +125,30 @@ const makeTranslate = (delta: RectangleBase) => ({ ofWin, rect, offset }: Move):
 function getInitialPositions(win: OpenFinWindow) {
     return WindowGroups.getGroup(win.groupUuid).map(moveFromOpenFinWindow);
 }
-function handleBoundsChanging(
-    win: OpenFinWindow,
-    e: any,
-    rawPayloadBounds: RectangleBase,
-    changeType: ChangeType,
-    treatBothChangedAsJustAResize: boolean = false
-): Move[] {
+function handleBoundsChanging(win: OpenFinWindow, rawBounds: RectangleBase, changeType: ChangeType): Move[] {
     const initialPositions: Move[] = getInitialPositions(win);
-    const leaderRectIndex = initialPositions.map(x => x.ofWin).indexOf(win);
-    let moves: Move[];
-    const startMove = moveFromOpenFinWindow(win); //Corrected
-    const start = startMove.rect;
-    const { offset } = startMove;
-    const end = normalizeExternalBounds(rawPayloadBounds, offset); //Corrected
+    let moves = initialPositions;
+    const delta = getLeaderDelta(win, rawBounds);
     switch (changeType) {
         case ChangeType.POSITION:
-            moves = handleMoveOnly(start, end, initialPositions);
+            moves = handleMoveOnly(delta, initialPositions);
             break;
         case ChangeType.SIZE:
-            moves = handleResizeOnly(leaderRectIndex, startMove, end, initialPositions);
+            moves = handleResizeOnly(win, delta);
             break;
         case ChangeType.POSITION_AND_SIZE:
-            const delta = start.delta(end);
+            const resized = (delta.width || delta.height);
             const xShift = delta.x ? delta.x + delta.width : 0;
             const yShift = delta.y ? delta.y + delta.height : 0;
-            const shift = { x: xShift, y: yShift, width: 0, height: 0 };
-            const resizeDelta = {x: delta.x - xShift, y: delta.y - yShift, width: delta.width, height: delta.height};
-            const resized = (delta.width || delta.height);
-            moves = resized
-                ? handleResizeOnly(leaderRectIndex, startMove, start.shift(resizeDelta), initialPositions)
-                : initialPositions;
             const moved = (xShift || yShift);
-            //This flag is here because sometimes the runtime lies and says we moved on a resize
-            //This flag should always be set to true when relying on runtime events. It should be false on api moves.
-            //Setting it to false on runtime events can cause a growing window bug.
-            moves = moved && !treatBothChangedAsJustAResize
-                ? handleMoveOnly(start, start.shift(shift), moves)
-                : moves;
-
+            if (resized) {
```

Comment:
Do we no longer hit this on user moves? this used to cause growing windows in certain conditions

Reply:
Right, the changetype is being set to either 0 or 1 (move or resize) so this can only be hit on API calls.